packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -249,6 +252,44 @@ contract L2Genesis is Deployer {
         _setImplementationCode(Predeploys.L2_TO_L1_MESSAGE_PASSER);
     }
 
+    function setSoulGasToken() public {
+        address impl = _setImplementationCode(Predeploys.SOUL_GAS_TOKEN);
+
+        if (Constants.IS_SOUL_QKC) {
```

Comment:
Maybe we should change the name "isSoulQKC" to "backedByNative"?

Reply:
Good idea, I renamed it to `IS_SOUL_BACKED_BY_NATIVE` so that the context is clear.